.github/workflows/workflow.yaml

```diff
@@ -317,6 +317,59 @@ jobs:
             done
           done
 
+  benchmark:
+    name: "Benchmark"
+    if: github.event_name == 'pull_request'
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v3
+
+      - name: Download release benchmark file
+        uses: robinraju/release-downloader@v1.6
+        id: get-latest-benchmark
+        continue-on-error: true
```

Comment:
do we want to continue?

Reply:
We are not really executing rest of the steps if the file is missing.

If we dont continue on error, the whole job fails and I think it be best if we not fail on missing previous benchmark file, just skip all the steps of the workflow if release benchmark is missing